lib/StripeResource.js

```diff
@@ -134,13 +134,26 @@ StripeResource.prototype = {
       res.on('end', function() {
         var headers = res.headers || {};
 
+        // For convenience, make Request-Id easily accessible on
+        // lastResponse.
+        res.requestId = headers['request-id'];
+
+        var responseEvent = utils.removeEmpty({
+          api_version: req._requestEvent.api_version,
```

Comment:
Oh crazy!

Well I'm glad that we found a problem, but I'm not sure the headers are lower cased from Stripe. Try a Curl with `-i`:

```
$ curl https://api.stripe.com/v1/charges/ch_1ApLEe2eZvKYlo2CDgGoqDnH -u sk_test_BQokikJOvBiI2HlWgH4olfQ2: -i
HTTP/1.1 200 OK
Server: nginx
Date: Thu, 10 Aug 2017 18:18:41 GMT
Content-Type: application/json
Content-Length: 1781
Connection: keep-alive
Access-Control-Allow-Credentials: true
Access-Control-Allow-Methods: GET, POST, HEAD, OPTIONS, DELETE
Access-Control-Allow-Origin: *
Access-Control-Max-Age: 300
Cache-Control: no-cache, no-store
Request-Id: req_PAT5qZFg3PGNFx
Stripe-Version: 2017-06-05
Strict-Transport-Security: max-age=31556926; includeSubDomains

{
  "id": "ch_1ApLEe2eZvKYlo2CDgGoqDnH",
  "object": "charge",
  "amount": 4000,
...
```